app/main/lib/similarity.py

```diff
@@ -133,7 +133,7 @@ def callback_add_item(item, similarity_type):
     if function:
         response = function(item)
         app.logger.info(f"[Alegre Similarity] CallbackAddItem: [Item {item}, Similarity type: {similarity_type}] Response looks like {response}")
-        return response
+        return {"item": response}
```

Comment:
Question from someone not familiarized enough with this code: I see that this condition applies to all types of media (audio, image, video and text)... just making sure that fixing it for text here this way won't break the other media types.

Reply:
Yes - I checked with an image and does not appear to have an adverse effect - in fact, I don't believe there is *any* case where we run through this code-path with non-text at this time?